### PR TITLE
Update dyaml dependency: v0.7.1 -> v0.8.0

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: source/agora/common/Config.d [hunk 1]
```diff
@@ -177,8 +177,11 @@ public Config parseConfigFile (CommandLine cmdln)
 
     Node root = Loader.fromFile(cmdln.config_path).load();
 
-    enforce("network" in root && root["network"].isSequence,
-        "The 'network' section is mandatory and must specify at least one peer");
+    if (auto node = "network" in root)
```
Comment (AndrejMitrovic) [inline source/agora/common/Config.d:180]: Unrelated change?

Comment (Geod24) [reply source/agora/common/Config.d:180]: No they removed the property, look at the `enforce`.
Since I'm changing that code I might as well remove the double lookup.

Comment (AndrejMitrovic) [reply source/agora/common/Config.d:180]: oh sorry I misread the dependency

Comment (Geod24) [reply source/agora/common/Config.d:180]: https://github.com/dlang-community/D-YAML/commit/b63ea1aaae1f366d325c58aa36b19df4ec6c1130#r34022924

**Review by AndrejMitrovic — APPROVED** [review]
